index.js

```diff
@@ -1,20 +1,28 @@
 import process from 'node:process';
 import execa from 'execa';
 
-export async function runAppleScript(script) {
+const DEFAULT_OPTIONS = {
+	humanReadableOutput: true,
+};
+
+export async function runAppleScript(script, options = DEFAULT_OPTIONS) {
```

Comment:
```suggestion
export async function runAppleScript(script, {humanReadableOutput = true} = {}) {
```

Reply:
I believe the original approach ends up being more readable and easy to extend.
Nevertheless, I updated it as suggested in [da05ff8](https://github.com/sindresorhus/run-applescript/pull/14/commits/da05ff82061133e87977cfc2af0782a18c46356f), since I don't think there will be much more options to add.